### ISSUE TITLE
PYFA won't run at all on WIN 10

### ISSUE THREAD

**Issue by eliasmarobin** [pyfa-org/Pyfa#2147]
## Bug Report
After .exe installation or .zip decompress, double clicking the pyfa.exe or using the shortcut provided with the installation the pyfa.exe pops up momentarily on the taskmanager and vanishes without any message.

### Expected behavior:
The program to run.

### Actual behavior:
Program instantly closes without any screen or any error. Not even the .pyfa folder actually exist and I created it manually but nothing happened either.

### Detailed steps to reproduce:
Install it and try to run, nothing shows up.

### Fits involved in EFT format (Edit > To Clipboard > EFT):
None

### Release or development git branch? Please note the release version or commit hash:
2.18.1

### Operating system and version (eg: Windows 10, OS X 10.9, OS X 10.11, Ubuntu 16.10):
Windows 10

### Other relevant information:
Can't find anything related to the crash, nothing on windows event viewer, nothng on .pyfa folder. Does not know if there is any requirements to run the program itself other than the .exe installation file.

**Comment by navexor** [comment]
The same problem. The program is even not appearing on the taskbar. The only Windows "beep" and nothing else.

**Comment by DarkFenX** [comment]
Unsure what could cause it tbh.

1) Try opening command prompt, executing `cd <pyfa folder>` and running `pyfa.exe` in there, then see if anything gets printed out.
2) Tell me if your system is 32 bit or 64 bit
3) In legacy control panel, open regional settings. There you should have "Format" selection with huge drop-down list. Tell me what's your current setting

**Comment by navexor** [comment]
1. Nothing in the console. Just beep.
![image](https://user-images.githubusercontent.com/3763295/78557413-6ad3c380-7819-11ea-8412-cc7e2e07a1df.png)
2. 64 bit
3. Not sure if it helps, I have Russian version
![image](https://user-images.githubusercontent.com/3763295/78557625-ca31d380-7819-11ea-8e62-8d30b679f7bf.png)
4. version: v2.18.1

**Comment by DarkFenX** [comment]
3. Try setting English (US) temporarily and see if it helps. There were bunch of fixes and workarounds related to it. So it might help, who knows.
4. Please re-read the question. Last version which *worked* for you, not the broken one.

**Comment by navexor** [comment]
3. Setting English didn't help
![image](https://user-images.githubusercontent.com/3763295/78558196-da967e00-781a-11ea-986a-a5675006bf26.png)
4. It was a long time ago when I last used the pyfa. 2-3 years ago maybe. But OK, I'll try older versions later one by one.

**Comment by DarkFenX** [comment]
Okay, so format stuff is unrelated. Do you have any anti-virus monitors by any chance? Could you try disabling them and seeing if it helps?

Do you have DEP (data execution prevention) enabled (also unsure if it is related, but this setting affected functionality of several programs for me in the past)?

**Comment by navexor** [comment]
No. Only clear Win 10 64bit with built-in standard antivirus. No known DEP enabled. I even don't know what is that.

**Comment by hozayn** [comment]
Right-click on dowloaded exe or zip -> properties -> then select "unblock" option. 

Example pic:
[https://winaero.com/blog/wp-content/uploads/2015/10/Windows-10-unblock-a-file.png](url)

Now you can install/extract without issue i hope.

**Comment by navexor** [comment]
@hozayn Thank you! pyfa is finally working.

**Comment by DarkFenX** [comment]
Wow, this is new, installer has never been completely blocked before, only warning has been shown...

**Comment by marcos-alfredo** [comment]
I'm having the same problem as described above.
Tried all solutions, inclusive the DEP and checking if the exe is blocked. Nothing worked so far.

I've tried standalone and installation, no error or log files, nothing in events viewer, no .pyfa folder in my user profile.

Not sure what else to do try now.

**Comment by qJackDodgersp** [comment]
Marcos, are you Brazilian?

**Comment by marcos-alfredo** [comment]
I am, yeah.

I looked over your profile and found your bug report.
Freaking Warsaw!
The process name was core.exe for me. I did notice it yesterday but thought nothing of it.

When I went to uninstall it they asked for a captcha!
Never seen that before.

It's working now with the latest release, thanks for the help man!

Valeu!
